reinforcement_learning/testrunner.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import itertools
+import json
+import os
+import subprocess
+import sys
+import time
+import yaml
+try:
+    import pandas as pd
+    import papermill
+    from tabulate import tabulate
+except ImportError:
+    sys.exit("""Some libraries are missing. Please install them by running `pip install -r test_requirements.txt`.""")
+
+# CONSTANTS
+TEST_NOTEBOOKS_FILE = 'testnotebooks.yml'
+TEST_CONFIG_FILE = 'testnotebooks.yml'
```

Comment:
should be `testconfig.yml`?

Reply:
Fixed, and running the test E2E on AWS to see the results.